ChatQnA/chatqna.py

```diff
@@ -58,11 +58,17 @@ def generate_rag_prompt(question, documents):
 LLM_SERVER_HOST_IP = os.getenv("LLM_SERVER_HOST_IP", "0.0.0.0")
 LLM_SERVER_PORT = int(os.getenv("LLM_SERVER_PORT", 80))
 LLM_MODEL = os.getenv("LLM_MODEL", "Intel/neural-chat-7b-v3-3")
+EMBEDDINGS_MODEL_ID = os.getenv("EMBEDDINGS_MODEL_ID", "BAAI/bge-base-en-v1.5")
+EMBEDDINGS_USE_VLLM = os.getenv("EMBEDDINGS_USE_VLLM", "false")
```

Comment:
We are refactoring the GenAIComps embedding code to ensure full compatibility with the OpenAI format. With this update, there is no need to differentiate vLLM in this context. All embedding requests should use the endpoint `/v1/embeddings` and include the `model` parameter.

Reply:
thanks

Reply:
> We are refactoring the GenAIComps embedding code to ensure full compatibility with the OpenAI format. With this update, there is no need to differentiate vLLM in this context. All embedding requests should use the endpoint `/v1/embeddings` and include the `model` parameter.

I found following 3 PRs:
* https://github.com/opea-project/GenAIComps/pull/289
* https://github.com/opea-project/GenAIComps/pull/275
* https://github.com/opea-project/GenAIComps/pull/1035

Are there (going to be) more?